docs/README.md

```diff
@@ -56,7 +56,37 @@ Each chapter must link a file: internal links to anchors within files do not
 work. There is an [open issue](https://github.com/rust-lang/mdBook/issues/167)
 to fix this behavior.
 
+### TLA+ syntax highlighting
+
+We provide a custom TLA+ syntax highlighting plugin for [highlight.js][] in `highlightjs-tlaplus`.
+
+#### Building
+
+To build, check out the main highlight.js into a separate directory.
+Symlink the TLA+ language definition into the highlight.js `extra` directory:
+
+```sh
+$ git clone https://github.com/highlightjs/highlight.js.git
+$ cd highlight.js
+$ ln -s path/to/apalache/docs/highlightjs-tlaplus/ ./extra/
+```
+
+Next, fetch the highlight.js dependencies and build:
+
+```sh
+$ npm install
+$ node ./tools/build.js -t cdn
+```
+
+The minified CDN distributable is now available in `highlightjs-tlaplus/dist/`.
+
+For more packaging information, see the highlight.js [packaging instructions][].
```

Comment:
In principle, that'd be great! I just don't really know how to pull it off, since none of these are "hard" dependencies for Apalache.

Would we also automate the `git clone` of highlight.js? If yes, where would we clone to, `/tmp`? If no, we'd need to pass the path to highlight.js in an env var.
Also, neither `npm` nor Node.js are Apalache dependencies, so the Makefile target would probably fail without further prep.

If you have any ideas here, I'm all ears :)

Reply:
I also thought about simply hosting `highlightjs-tlaplus` in a separate repo, but then we'd need sync the built plugin there with the Apalache repo.

Reply:
Sounds like more trouble than it’s worth at this point. I think this is a great solution for the time being!